Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -9,9 +9,19 @@ Better define an inner function to find the next minimum in the array.
 
 */
 
-func selectionSort(array: [Int]) -> [Int] {
-    // You may declare array argument with var keyword so that it is copied.
-
+func selectionSort(var array: [Int]) -> [Int] {
+    
+    for index in array.indices {
+        var indexOfMinimumValue = index
+        for innerIndex in index + 1..<array.count {
```

Comment:
I opted for a inner loop here, but maybe extracting this `indexOfMinimumValue` logic into a local function can be easier to follow.